backend/app/core/config.py

```
@@ -0,0 +1,78 @@
+    APP_VERSION: str = "0.1.0"
+    APP_ENV: str = "development"
+
+    HOST: str = "0.0.0.0"
+    PORT: int = 8000
+    RELOAD: bool = False
+
+    CORS_ORIGINS: str = "*"
+
+    DATABASE_URL: str = "postgresql+asyncpg://app_user:app_password@localhost:5432/app_db"
+    REDIS_URL: str = "redis://localhost:6379/0"
+    JWT_SECRET: str = "change_me_to_a_long_random_secret"
```

Ruff (0.15.4)

[error] Possible hardcoded password assigned to: "JWT_SECRET"

(S105)